src/index.js

```diff
@@ -86,3 +85,8 @@ export function registerApp() {
 
 	AppRegistry.registerComponent( 'gutenberg', () => RootComponent );
 }
+
+export function setupApp() {
```

Comment:
Nit, let's call this `bootstrapEditor` or something. The "App" moniker starts to feel too vague, WDYT? :)

Reply:
Agreed 👍